packages/amplify-provider-awscloudformation/lib/push-resources.js

```diff
@@ -475,6 +485,54 @@ function updateIdPRolesInNestedStack(context, nestedStack, authResourceName) {
   Object.assign(nestedStack.Resources, idpUpdateRoleCfn);
 }
 
+ function packageLayer(context,resource){
```

Comment:
this function should really be in the function category plugin. Ideally it wouldn't depend directly on a category, but it already depends on auth and predictions, so may as well add functions